Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -845,6 +846,32 @@ private void LoadModuleInFrame()
             }
         }
 
+
+        private void HomeLogoAndSiteIcons()
+        {
+            string PortalRoot = PageManager.GetPortalRoot(PortalSettings.Current.PortalId);
+            PortalRoot = PortalRoot + "/";
+            string SocialSharingLogo = PortalRoot + PortalController.GetPortalSetting("SocialSharingLogo", PortalSettings.Current.PortalId, "", PortalSettings.CultureCode);
+            string HomeScreenIcon = PortalRoot + PortalController.GetPortalSetting("HomeScreenIcon", PortalSettings.Current.PortalId, "", PortalSettings.CultureCode);           
+            Literal lt = new Literal();
+            if (!string.IsNullOrEmpty(SocialSharingLogo))
+            {
+                lt.Text += "<meta property=\"og:image\" content=\"" + SocialSharingLogo + "\">";
+                lt.Text += "<meta property=\"og:image:type\" content=\"image/png\">";
+                lt.Text += "<meta property=\"og:image:width\" content=\"1024\">";
```

Comment:
Why is dimensions hard coded to 1024 x 1024? Is this what we're requiring at upload?

Reply:
How do we override the og:image? In case of a blog post?